### DISCUSSION TITLE
Can't get lldb to work (breakpoints always get rejected)

### DISCUSSION THREAD

**Discussion by lsig** [mfussenegger/nvim-dap#817] [Q&A]
Hi, 
I have been trying to get the lldb dap to work for hours and I can't seem to get the breakpoints to work, they always get rejected. I have looked at the troubleshooting and added debug symbols to the binary with the -O0 flag (also tried -g)  but it doesn't do the trick. 

Here is my config; 
![Screenshot 2023-01-11 at 22 55 10](https://user-images.githubusercontent.com/89947902/211935681-e27d7770-a71a-45d4-a7cd-6ae8b02f71e8.jpg)

Here is the dap.log; 
![Screenshot 2023-01-11 at 22 56 57](https://user-images.githubusercontent.com/89947902/211935979-19d89bb1-4930-411e-8137-16b2f4d996f4.jpg)

mason-dap config;
![Screenshot 2023-01-11 at 22 58 33](https://user-images.githubusercontent.com/89947902/211936238-1a49c4f7-4768-443c-86ae-b8d3d7a005d9.jpg)

Any ideas?

**Comment by mfussenegger** [comment]
The setup looks correct so it is likely an issue with the debug symbols.
It could be that you're compiling the binary with a newer DWARF standard than what lldb supports.

You could try setting it explicitly, for example with clang: `clang -gdwarf-4 foo.c -o foo`

And just to be sure: The path to program needs to point to the compiled binary, not the source file

**Reply by lsig** [reply]
I seem to be getting closer but now when I try to run DapContinue I get Error on launch: Permission denied (I'm on MacOS). Do you know where I have to change the rights to allow the debugger to attach? 
Dap.log (in trace mode) 
![Screenshot 2023-01-12 at 19 43 11](https://user-images.githubusercontent.com/89947902/212165353-bb3f2a00-6485-4107-899b-b90cbbae81da.jpg)
